common/src/main/java/com/wynntils/functions/TestFunction.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.functions;
+
+import com.wynntils.core.functions.Function;
+
+public class TestFunction extends Function<Integer> {
+    @Override
+    public Integer getValue(String argument) {
+        return 42;
```

Comment:
There is no need for this function to exist. We have a real world example.

Reply:
I'll remove it as soon as we have real functions. But I'm not sure what real world example you are thinking of..? I do need some kind of test function in this PR, so I'm planning on removing this in a follow-up when we start to port over legacy functions.

Reply:
@magicus what is this then? common/src/main/java/com/wynntils/functions/WorldNameFunction.java

Reply:
And with a real world example I am referring to an actual implementation as we might expect in production

Reply:
Ok, I'll port over a handful of "info variables" from legacy, so we can see this API in action. It's a fair point, which I'm usually trying to make. ;-)

Reply:
We lack the source data for most of our info variables. I made a list at https://github.com/Wynntils/Artemis/issues/301 for keeping track of what is still left to do. I implemented two categories of functions: those that derive their value from Minecraft, and those that derive their value from the general environment (basically, the JVM).

Reply:
Ok, I think I got this covered for now. If anything, we should perhaps try to get some ActiveFunctions as well. I'll see if there is any I can implement without far too much work.

Reply:
For some reason my comment disappear regarding the already present worldnamefunction :?